src/cljs/ataru/hakija/application_handlers.cljs

```diff
@@ -219,24 +219,41 @@
   (let [options (get-in db [:application :answers (keyword (:id field-descriptor)) :options])]
     (set-followups-visibility db selected-hakukohteet-and-ryhmat field-descriptor #(get options (:value %)))))
 
+
+(defn- set-option-visibility [db [index option] field-descriptor selected-hakukohteet-and-ryhmat]
+  (if-let [belongs-to (seq (concat (:belongs-to-hakukohderyhma option)
+                                   (:belongs-to-hakukohteet option)))]
+    (assoc-in db [:application :ui (keyword (:id field-descriptor)) index :hide?] (not (not-empty (clojure.set/intersection
```

Comment:
`:hide?` sijaan vois olla `:visible?` niin olis samoin päin kun normi näkyvyydessä. Toki sit tuolla else haarassa pitää kans asettaa `:visible?` todeksi.

`(not (not-empty ...` sijaan vois olla `empty?` :)